### PR TITLE
Add command guides

### PR DESCRIPTION
**Pull request by exoRift**

### CODE REVIEW

File: lib/agent/index.js [hunk 1]
```diff
@@ -453,6 +507,16 @@ class Agent {
...
   }
 
+  /**
+   * Format handler data by turning existing single supply into arrays if necessary
+   * @private
+   * @param   {Array<Command|Replacer|ReactCommand>|Command|Replacer|ReactCommand} data The handler data
+   * @returns {Array<Command|Replacer|ReactCommand>}                                    The formatted data
+   */
+  _formatHandlerData (data) {
+    if (data) return Array.isArray(data) ? data : [data]
```
Comment (exoRift) [inline lib/agent/index.js:517]: @alex-taxiera Would it be better for code clarity's sake to add `else return undefined`?

File: lib/structures/command/index.js [hunk 2]
```diff
@@ -54,15 +63,28 @@ class Command {
...
+     * @prop {Number}     guide.color  An integer color for the sidebar of the command guide
+     * @prop {Object[]}   guide.fields A list of field objects displayed in the guide
+     * @prop {Boolean}    restricted   Whether or not this command is restricted to admin only
+     * @prop {Boolean}    guildOnly    Whether or not this command can only be triggered in guilds as opposed to DMs
+     * @prop {Number}     authLevel    The minimum auth level required to execute this command
      */
     this.options = {
-      args,
+      args: args.map((arg) => {
+        arg.delim = arg.delim || ' '
```
Comment (alex-taxiera) [inline lib/structures/command/index.js:77]: this mapping is duplicated in await

File: lib/structures/replacer/index.js [hunk 3]
```diff
@@ -35,7 +35,13 @@ class Replacer {
      * @prop {Argument[]} args The arguments for the replacer
      */
     this.options = {
-      args
+      args: args.map((arg) => {
+        arg.delim = arg.delim || ' '
```
Comment (alex-taxiera) [inline lib/structures/replacer/index.js:39]: and replacer

File: lib/agent/index.js [hunk 4]
```diff
@@ -275,7 +331,7 @@ class Agent {
       const {
         desc,
         serverCode,
-        color = 0,
+        color = 33023, // Cyclone blue
```
Comment (alex-taxiera) [inline lib/agent/index.js:334]: don't you have a place to declare constants?

**Pull request merged**